app/src/hooks/use-keyboard-shortcut-handler.ts

```
@@ -0,0 +1,50 @@
+export function useKeyboardShortcutHandler() {
+  const navigate = useNavigate();
+  
+  return useCallback((action: keyof KeyboardShortcutsConfig, event: KeyboardEvent) => {
+    switch (action) {
+      case "newChat":
+        navigate({ to: "/chat/new" });
+        break;
+      case "search":
+        // Trigger search modal or focus search input
+        const searchInput = document.querySelector<HTMLInputElement>('[data-search-input]') || 
+                           document.querySelector<HTMLInputElement>('[data-testid="search-input"]');
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)